apps/experiments/views/experiment.py

```diff
@@ -1216,6 +1225,43 @@ def _experiment_chat_ui(request, embedded=False):
     )
 
 
+def _get_available_languages_for_chat(chat_id, clear_cache=False):
```

Comment:
Can't we use the values from `chat.translated_languages` instead of getting it from the `ChatMessage` objects?